test/anvil/anvil-global-setup.ts

```diff
@@ -102,6 +102,23 @@ export async function stopAnvilForks() {
     runningForks = {};
 }
 
+// Stop a specific anvil fork
+export async function stopAnvilFork(
```

Comment:
It was only really an issue here because the previous tests were removing enough liquidity from pools that made future swap paths impractical.
I do think there is likely a better solution to what I found. Originally I tried [reset](https://viem.sh/docs/actions/test/reset#reset) which I thought would be ideal but it really slows down the tests. Probably worth making an issue to investigate/understand this better as we should really try to optimise for speed and provider calls if possible?

Reply:
I created #291 to keep track of this ✅